src/GraphQL/Language/AST/Variables.cs

```diff
@@ -5,10 +5,16 @@
 
 namespace GraphQL.Language.AST
 {
+    /// <summary>
+    /// Contains a list of variables (name &amp; value tuples) that have been gathered from the document and attached <see cref="Inputs"/>.
```

Comment:
attached **to** ?

Reply:
Not in this case.  So it's not "attaching a rope to a tree"  but rather more like "Please open the attached files."

Reply:
ok